### ISSUE TITLE
CloudManager.create_server() ignores `Storage.address` attribute.

### ISSUE THREAD

**Issue by anton-molyboha** [UpCloudLtd/upcloud-python-api#86]
I expect that the code below would create a server with its only storage attached as a virtio device. Instead, the created server has the storage attached as an IDE device.
```python
        _manager = upcloud_api.CloudManager(login, password)
        _manager.authenticate()
        storage = _manager.get_storage(uuid_of_the_template)
        storage.address = "virtio:0"
        srv = upcloud_api.Server(
            plan = "1xCPU-2GB",
            hostname = "testpackage{:06}".format(random.randrange(1000000)),
            zone = storage.zone,
            storage_devices = [storage],
        )
        _manager.create_server(srv)
```

**Comment by Darep** [comment]
Thanks for reporting the problem! Were you able to get it working already?

We tried the snippet you provided* with a custom template, a public template and a storage from another server, and all were attached as virtio on our servers. Not sure where the problem might be… Which zone are you working in?

Could also try to define the fields for the storage explicitly inside the "storage_devices" instead of passing a Storage object, and also `address = "virtio"` without the :0

*) note: we added "delattr" to your snippet just before the .create_server() call to fix the ["unknown attribute"-error](https://github.com/UpCloudLtd/upcloud-python-api/issues/84)

```python
delattr(srv.storage_devices[0], 'zone')
```
